docs/resources/Guild.md

```diff
@@ -667,6 +667,9 @@ Modify a guild's settings. Requires the `MANAGE_GUILD` permission. Returns the u
 > info
 > This endpoint supports the `X-Audit-Log-Reason` header.
 
+> warn
+> Attempting to omit the `COMMUNITY` feature when it was previously present will fail unless the bot has administrator, Attempting to add this feature when it was not present beforehand will also fail for the same reason.
```

Comment:
```suggestion
> Attempting to omit the `COMMUNITY` feature when it was previously present will fail, unless the bot has `ADMINIATRATOR` permission. Attempting to add this feature when it was not present beforehand, will also fail for the same reason.
```

Reply:
pluralise permission

Reply:
Would make more sense as either "has `ADMINISTRATOR`" or "has the `ADMINISTRATOR` permission" I think

Reply:
Could also shorten it and include a reference (not yet including any permission wording changes):

```suggestion
> Attempting to add or remove the `COMMUNITY` [guild feature](#DOCS_RESOURCES_GUILD/guild-object-guild-features) will fail unless the bot has `ADMINISTRATOR` permission.
```